server/app/assets/javascripts/preview.ts

```diff
@@ -463,11 +463,23 @@ class PreviewController {
 
     items.forEach((item) => {
       const listItem = document.createElement('li')
-      listItem.textContent = item
+      listItem.appendChild(PreviewController.parseMarkdown(item))
       listTag.appendChild(listItem)
     })
     return listTag
   }
+
+  private static parseMarkdown(currentLine: string): Element {
+    const md = new MarkdownIt()
+    const parser = new DOMParser()
+    let html
+    if (currentLine[0] == this.headerIndicator) {
```

Comment:
(blocking) Please add a guard clause to make sure `currentLine[0]` can't meltdown the markdown if currentLine is null or an empty string.

Reply:
done